src/MessageContainer.tsx

```diff
@@ -194,14 +198,16 @@ export default class MessageContainer<
     )
   }
 
+  renderTypingIndicator = () => {
```

Comment:
Is that possible to provide a way to let user of `renderTypingIndicator` implement it by itself?
Meaning providing a renderTypingIndicator props.
Thanks

Reply:
Hey, I can't! The problem is that the animation and chat bubble completely depend on the size of the indicator. There's no option to use flexbox, so it needed to be manually aligned. The easiest way for them to provide their own chat indicator is by using `renderFooter` and plumbing their own indicator, state and animation. This also maintains backwards compatibility with all of the current users who already provided their indicator this way and if I tried to do anything else it would be a breaking change.